README.md

```diff
@@ -20,7 +20,146 @@ testing framework runner.
+technologies (e.g. vagrant, docker).
+
+
+### Kitchen Proxy Driver
+The proxy driver is included with the base test-kitchen installation.
+
+When testing on a pre-deployed system using the proxy driver, it was necessary
+to create a symlink of the gem command on the target system to
+`/opt/chef/embedded/bin/gem`.
+```
+sudo ln -s /usr/bin/gem /opt/chef/embedded/bin/gem
+```
+
+## Kitchen Provisioners
+
+### Ansible Playbook
+For this repo, we want to use the ansible playbook provisioner. So, it is
+necessary to install it as well.
+```
+gem install kitchen-ansible
+```
+
+## Execution
+Now that everything is in place, we can use test kitchen to run the ansible
+playbook(s) to set up the environment and then use busser-bats to test it.
+```
+KITCHEN_SUT=<hostname or ip address of SUT> kitchen converge
+KITCHEN_SUT=<hostname or ip address of SUT> kitchen verify
+```
+
+### Variables
+#### Environment variable for kitchen
+__This KITCHEN_SUT variable is required.__
+The .kitchen.yml file uses the environment variable `KITCHEN_SUT` to identify
+the pre-deployed host that kitchen should used for the provisioner and
+verifier.
+
+#### Testing variables for bats
+In order to provide `bats` with the variables needed to perform the desired
+OpenStack validation, they must be updated in the
+`tests/integration/default/bats/variables.bash` file. Since `bats` is run
+directly on the SUT it cannot inherit environment variables from `kitchen.
+Therefore, they are stored in the `variables.bash` file which is copied to the
+SUT and loaded during the test execution phase.
+
+__The testing variables MUST be updated to reflect your target environment.__
+
+## Example setup
+### Ubuntu 14.04 Trusty
+The following example setup assumes the use of Ubuntu 14.04 (Trusty-Tahr) as
+the test runner for all of this to be installed on. These commands are
+assembled in one place below from those given above for your convenience.
+```
+sudo apt-get update
+sudo apt-get install -y git-core
+sudo apt-get install -y python3 python3-dev
+sudo apt-get install -y libssl-dev libreadline-dev zlib1g-dev  # these libraries are needed to compile ruby
+sudo apt-get install -y virtualbox
+curl "https://bootstrap.pypa.io/get-pip.py" -o "get-pip.py"
+sudo python get-pip.py
+sudo pip install --upgrade virtualenv
+git clone https://github.com/rbenv/rbenv.git ~/.rbenv
+echo 'export PATH="$HOME/.rbenv/bin:$PATH"' >> ~/.bashrc
+~/.rbenv/bin/rbenv init
+echo 'eval "$(rbenv init -)"' >> ~/.bashrc
+source ~/.bashrc
+curl -fsSL https://github.com/rbenv/rbenv-installer/raw/master/bin/rbenv-doctor | bash
+mkdir -p ~/.rbenv/plugins
+git clone https://github.com/rbenv/ruby-build.git ~/.rbenv/plugins/ruby-build
+rbenv install 2.4.3
+rbenv rehash
+rbenv global  2.4.3
+git clone https://github.com/johnduarte/kitchen-ansible-example.git
+cd kitchen-ansible-example
+virtualenv --python python3 venv
+source venv/bin/activate
+gem install test-kitchen
+gem install kitchen-ansible
+```
```

Comment:
`kitchen` looks for `gem` in a hardcoded link `/opt/chef/embedded/bin/gem` as you pointed out in a working session, so I think the below 2 lines need to be added for `kitchen verify` to work:
`sudo mkdir -p /opt/chef/embedded/bin`
`sudo ln -s /usr/bin/gem /opt/chef/embedded/bin/gem`